File: FINAL/4 - Algorismes/02 - P29094 - Posició del màxim/main.py
from yogi import *

m, n = scan(int), scan(int)
while m is not None and n is not None:
    v = []
    for i in range(0, n):
        v.append(read(float))
    print(posicio_maxim(v, m))
    m, n = scan(int), scan(int)

File: FINAL/4 - Algorismes/02 - P29094 - Posició del màxim/main.cc
#include <iostream>
#include <vector>
using namespace std;

int posicio_maxim(const vector<double>& v, int m);

int main()
{
    int m, n;
    while (cin >> m >> n) {
        vector<double> V(n);
        for (int i = 0; i < n; ++i)
            cin >> V[i];
        cout << posicio_maxim(V, m) << endl;
    }
    return 0;
}
